main.py

```diff
@@ -161,12 +161,8 @@ def generate_beer_list(travel_df, beers_df):
     s = ""
     s += f"Collected {count_beer(travel_df)} beer types:\n"
     for row in travel_df.itertuples():
-        if type(beers_df.loc[row.brewery_id]["name"]) == str:
-            name = beers_df.loc[row.brewery_id]["name"]
+        for name in beers_df.name.loc[[row.brewery_id]].values:
```

Comment:
Is it brewery name or beer name?
Also, I think
`beers_df.loc[[id],name].value`

Reply:
Beer name

Reply:
User beer_name variable name for better readeability

Reply:
Fixed, in this and all other cases.